### PR TITLE
Partial tests for Community Modules package

### PR DESCRIPTION
**Pull request by Cortey**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**
#2117

### CODE REVIEW

File: internal/communitymodules/merge_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package communitymodules
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestMergeRowMaps(t *testing.T) {
```
Comment (anoipm) [inline internal/communitymodules/merge_test.go:8]: I can't see tests for other cases i.e:
- more than one key inside `moduleMap`,
- merging maps with some different keys.

Now such data are probably impossible but we don't know about future. Tested module is generic library that should support such cases.

File: internal/communitymodules/render_test.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package communitymodules
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestRender(t *testing.T) {
```
Comment (anoipm) [inline internal/communitymodules/render_test.go:8]: What if `convertRow...` doesn't work for `moduleMap` with not one row?

File: internal/communitymodules/merge_test.go (hunk 1 above)
Comment (Cortey) [reply internal/communitymodules/merge_test.go:8]: Added case covering those

File: internal/communitymodules/render_test.go (hunk 2 above)
Comment (Cortey) [reply internal/communitymodules/render_test.go:8]: Added cases for those

**Review by anoipm — APPROVED** [review]

**Pull request merged**